nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Analyze/NuGetContext.cs

```diff
@@ -37,12 +37,23 @@ public NuGetContext(string? currentDirectory = null, ILogger? logger = null)
             .Where(p => p.IsEnabled)
             .ToImmutableArray();
         Logger = logger ?? NullLogger.Instance;
-        TempPackageDirectory = Path.Combine(Path.GetTempPath(), ".dependabot", "packages");
+        TempPackageDirectory = Path.Combine(Path.GetTempPath(), $"dependabot-packages_{Guid.NewGuid():d}");
```

Comment:
[for my learning]: will we ever need to clean up the directories? Or do they get wiped as part of the job run?